docs/docs/Workspace/workspace-api.md

```diff
@@ -36,6 +36,51 @@ The **Python Code** tab displays code to interact with your flow's `.json` f
 The **Tweaks** tab displays the available parameters for your flow. Modifying the parameters changes the code parameters across all windows. For example, changing the **Chat Input** component's `input_value` will change that value across all API calls.
 
 
+## Send image files to your flow with the API
+
+Send image files to the Langflow API.
```

Comment:
This is pretty much a repeat of the title. Can you expand on this sentence with some information about why you would want to do this?

Reply:
Added "for AI analysis"